src/components/edit-drop-down-lists/edit-drop-down-lists-page-2.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { useContext } from "react";
+import { InfoContext } from "../new-print-page/printContext";
+import { useHistory, useParams } from "react-router-dom";
+import endpointUrl from "../../config";
+import "./edit-drop-down-list-page.css";
+
+const EditDropDownListsPage2 = () => {
+  const filterOptions = (value, listToDeleteFrom) => {
+    if (listToDeleteFrom == null) return;
```

Comment:
Use strict equality here ie: `===`

It means that `listToDeleteFrom` must equal `null` exactly. Otherwise, these would be true:

```
undefined == null //true
```